### PR TITLE
Moved default values down to Configuration which is lowest level, not Config

### PR DESCRIPTION
**Pull request by HowieG**
## 📥 Pull Request

**📘 Description**
Moved default values down to Configuration which is lowest level, not Config

**🎯 Goal**
Move default setting closest to where it's used

**🔍 Additional Context**
You can't set defaults in __init__ using a function e.g. environ.get(...)

**🧪 Testing**
Ran openai test handler v1

### CODE REVIEW

**Review by HowieG — COMMENTED** [review]
https://www.loom.com/share/fbd1eb661f09410b8de22c3e8d96cb12

File: agentops/client.py [hunk 1]
```diff
@@ -58,27 +59,15 @@ def __init__(self, api_key: Optional[str] = None,
...
-                                    max_queue_size)
+
+        try:
+            self.config = Configuration(api_key,
+                                        parent_key,
+                                        endpoint,
+                                        max_wait_time,
+                                        max_queue_size)
+        except ConfigurationError:
+            return
```
Comment (siyangqiu) [inline agentops/client.py:74]: You return but you aren't logging the error that resulted in the configuration error

File: agentops/meta_client.py [hunk 2]
```diff
@@ -45,8 +45,9 @@ def wrapper(self, *args, **kwargs):
         try:
             return method(self, *args, **kwargs)
         except Exception as e:
-            type(self).send_exception_to_server(e, self.config._api_key)
-            logging.warning(f"AgentOps: Error: {e}")
```
Comment (siyangqiu) [inline agentops/meta_client.py:49]: We want this logging

File: agentops/client.py [hunk 3]
```diff
@@ -32,20 +31,22 @@ class Client(metaclass=MetaClient):
...
+        parent_key (str, optional): Organization key to give visibility of all user sessions the user's organization.
         tags (List[str], optional): Tags for the sessions that can be used for grouping or 
             sorting later (e.g. ["GPT-4"]).
-        endpoint (str, optional): The endpoint for the AgentOps service. Defaults to 'https://api.agentops.ai'.
+        endpoint (str, optional): The endpoint for the AgentOps service.
         max_wait_time (int, optional): The maximum time to wait in milliseconds before flushing the queue. 
             Defaults to 1000.
         max_queue_size (int, optional): The maximum size of the event queue. Defaults to 100.
         override (bool): Whether to override and LLM calls to emit as events.
+        auto_start_session (bool): Whether to start a session automatically when the client is created.
```
Comment (siyangqiu) [inline agentops/client.py]: We should mention all default values.
I know this is unrelated to your change, but can you change max_wait_time's say "Defaults to 30,000 (30 seconds)"

**Review by siyangqiu — APPROVED** [review]
Some nits, but great!

**Pull request merged**